test/__snapshots__/v3-client/test/generated/v3_client/core/OpenAPI.ts.snap

```diff
@@ -28,18 +55,5 @@ export const OpenAPI: OpenAPIConfig = {
     USERNAME: undefined,
     VERSION: '1.0',
     WITH_CREDENTIALS: false,
-};
-
-export const mergeOpenApiConfig = <T extends TResult>(config: OpenAPIConfig, overrides: TConfig<T>) => {
```

Comment:
Why was this removed?

Reply:
It's never used. I added it for generics response type, and with that setting it will be still generated